recipes/nvmolkit/variants.yaml

```diff
@@ -0,0 +1,28 @@
+cuda_compiler:
+  - cuda-nvcc
+cuda_compiler_version:
+  - "12.8"  # Support Blackwell
+python:
+  - "3.11.* *_cpython"
+  - "3.12.* *_cpython"
+  - "3.13.* *_cp313"
+rdkit:
+  - "2024.9.6"
+  - "2025.3.1"
+
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+  rdkit:
+    min_pin: x.x.x
+    max_pin: x.x.x
+
+is_python_min:
+  - true
+  - false
+  - false
+
+zip_keys:
+  - - python
+    - is_python_min
```

Comment:
Just to understand, why you specified the cuda version and the python version in the `variants.yaml` file ? Tipically version supported by conda-forge are defined in https://github.com/conda-forge/conda-forge-pinning-feedstock, and packages can skip given version (for example, skipping python==3.10 builds by requiring an higher python version). 

I see why this may needed due to https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/5719, but do you think it would be possible to drop that for Python and cuda?

Reply:
OK thanks for the info, that pinning package helps me understand the recipe logic better. I've dropped what I can from the variants for now. The CUDA 12.6 default is ok with us, build should work fine if not quite optimal on modern architectures. I'd rather get our first release out with the standard compiler than worry about compatibility. 

 I tried to pin the python version in recipe.yaml instead but am getting `Non noarch packages should have python requirement without any version constraints.` 

I'm also getting failures in the default linux build, because the cuda compiler version isn't populated. Should I skip the non-CUDA build? Not sure how exactly in the rattler-build recipe format.

Reply:
OK I applied the recommended fix with a small syntax change. I also removed the python version limit. Given the default builds of conda-forge are only off by 1 from my initial support matrix, it's not hard to extend to all 4 python versions and makes things easier.

Thanks!